src/Sarif/ConsoleLogger.cs

```diff
@@ -242,7 +252,7 @@ private string FormatNotificationMessage(Notification notification)
                 case NotificationLevel.Warning:
                 case NotificationLevel.Note:
                     issueType = notification.Level.ToString();
-                    issueType = issueType.Substring(0, 1).ToLowerInvariant() + issueType.Substring(1);
+                    issueType = issueType.Substring(0, 1).ToUpperInvariant() + issueType.Substring(1);
```

Comment:
No. You're actually changing the _function_ of the code here. It will produce different output after this change.

Many Code Analysis warnings can be addressed mechanically: name changes, argument checks, etc. But any time you change the function of the code, you have to think more carefully about it.

For now, we'll confine ourselves to purely mechanical changes. Please revert this change.